### PR TITLE
Editorial proposed changes for cc section

### PR DESCRIPTION
**Pull request by mirjak**

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -684,22 +684,24 @@ which only occurs after persistent congestion is declared.
...
+the congestion window, sets the slow start threshold to the new
+congestion window, and then enters the recovery period.
 
 ## Recovery Period
 
 A recovery period is entered when loss or ECN-CE marking of a packet is
-detected.  A recovery period ends when a packet sent during the recovery period
-is acknowledged.  This is slightly different from TCP's definition of recovery,
-which ends when the lost packet that started recovery is acknowledged.
+detected in congestion aviodance but after the congestion wiondow and slow
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]:
```suggestion
detected in congestion avoidance but after the congestion window and slow
```

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -684,22 +684,24 @@ which only occurs after persistent congestion is declared.
...
+start threhold has been adapted.  A recovery period ends when a packet sent
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]:
```suggestion
start threshold has been adapted.  A recovery period ends when a packet sent
```

Comment (martinthomson) [inline draft-ietf-quic-recovery.md]: adapted how?

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -766,17 +768,18 @@ similar to a sender's response on a Retransmission Timeout (RTO) in TCP
 
 ## Pacing {#pacing}
 
-This document does not specify a pacer, but it is RECOMMENDED that a sender pace
+This document does not specify a pacer, but it is RECOMMENDED that a sender paces
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]:
```suggestion
This document does not specify a pacer, but it is RECOMMENDED that a sender pace
```

File: draft-ietf-quic-recovery.md (hunk 2 above)
Comment (ianswett) [reply draft-ietf-quic-recovery.md]: Addressed in #3505, PTAL.

**Pull request merged**